### DISCUSSION TITLE
Support for simultaneous Toast notifications

### DISCUSSION THREAD

**Discussion by efonsecab** [CommunityToolkit/Maui#939] [New Feature Discussions]
Apparently, the current implementation of Toast notifications only support displaying 1 message at a time
<img width="472" alt="image" src="https://user-images.githubusercontent.com/3481899/215225687-416f53d9-41ed-497e-890a-346e1e011400.png">

I would a feature similar to Blazored.Toast where multiple notifications are shown simultaneously.

**Comment by VladislavAntonyuk** [comment]
I don't think it is possible because Google allows only 1 instance of toast/Snackbar.
Blazor implementation is like notifications queue.

**Reply by efonsecab** [reply]
> I don't think it is possible because Google allows only 1 instance of toast/Snackbar. Blazor implementation is like notifications queue.

Do you mean not allowed by Google Play Store rules?

**Reply by VladislavAntonyuk** [reply]
not allowed by design.
There can be only 1 toast displayed at the same time.

**Comment by CliffAgius** [comment]
Closed and answered
